packages/SwingSet/src/devices/mailbox.js

```diff
@@ -130,16 +132,14 @@ export function buildMailboxStateMap(state = harden(new Map())) {
   }
 
   function populateFromData(data) {
-    if (state.size) {
-      throw new Error(`cannot populateFromData: outbox is not empty`);
-    }
+    assert(!state.size, X`cannot populateFromData: outbox is not empty`);
     for (const peer of Object.getOwnPropertyNames(data)) {
       const inout = getOrCreatePeer(peer);
-      const d = data[peer];
+      const dp = data[peer];
```

Comment:
If we've gone back to using `X` for `details`, maybe these could revert back to their original form. I'm happy to leave them as `dp` though.

Reply:
Reverting them all.